include/gz/utils/config.hh.in

```diff
@@ -1,16 +1,23 @@
 /* Config.hh. Generated by CMake for @PROJECT_NAME_NO_VERSION@. */
 
 /* Version number */
-#define IGNITION_UTILS_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
-#define IGNITION_UTILS_MINOR_VERSION ${PROJECT_VERSION_MINOR}
-#define IGNITION_UTILS_PATCH_VERSION ${PROJECT_VERSION_PATCH}
+#define GZ_UTILS_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
+#define GZ_UTILS_MINOR_VERSION ${PROJECT_VERSION_MINOR}
+#define GZ_UTILS_PATCH_VERSION ${PROJECT_VERSION_PATCH}
+#define IGNITION_UTILS_MAJOR_VERSION GZ_UTILS_MAJOR_VERSION
+#define IGNITION_UTILS_MINOR_VERSION GZ_UTILS_MINOR_VERSION
+#define IGNITION_UTILS_PATCH_VERSION GZ_UTILS_PATCH_VERSION
 
-#define IGNITION_UTILS_VERSION "${PROJECT_VERSION}"
-#define IGNITION_UTILS_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_UTILS_VERSION "${PROJECT_VERSION}"
+#define GZ_UTILS_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define IGNITION_UTILS_VERSION GZ_UTILS_VERSION
+#define IGNITION_UTILS_VERSION_FULL GZ_UTILS_VERSION_FULL
 
-#define IGNITION_UTILS_VERSION_NAMESPACE v${PROJECT_VERSION_MAJOR}
+#define GZ_UTILS_VERSION_NAMESPACE v${PROJECT_VERSION_MAJOR}
+#define IGNITION_UTILS_VERSION_NAMESPACE GZ_UTILS_VERSION_NAMESPACE
 
-#define IGNITION_UTILS_VERSION_HEADER "Ignition ${IGN_DESIGNATION}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2020 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
+#define GZ_UTILS_VERSION_HEADER "Ignition ${IGN_DESIGNATION}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2020 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```

Comment:
Should the `Ignition` change to `Gz`?

Reply:
I updated that `Ignition` has part of another PR series, see https://github.com/gazebosim/gz-gui/pull/398/files#diff-eb6b9b2cedd00e7c9a1bfa077a11ad2eefed50f2925c9f05c83cf0f3fdae01bd. I don't really mind where we do it, as long as we get rid of all `Ignition` occurrences by the end.